themes-default/slim/views/editShow.mako

```diff
@@ -56,15 +181,16 @@ const startVue = () => {
                             <label for="location">
                                 <span class="component-title">Show Location</span>
                                 <span class="component-desc">
-                                    <input type="hidden" name="indexername" id="form-indexername" value="${show.indexer_name}" />
-                                    <input type="hidden" name="seriesid" id="form-seriesid" value="${show.series_id}" />
-                                    <input type="text" name="location" id="location" value="${show._location}" class="form-control form-control-inline input-sm input350"/>
+                                    <input type="hidden" name="indexername" id="form-indexername" :value="indexerName" />
+                                    <input type="hidden" name="seriesid" id="form-seriesid" :value="seriesId" />
+                                    <file-browser name="location" ref="locationBtn" id="location" v-model="location" class="form-control form-control-inline input-sm input350"></file-browser>
```

Comment:
@OmgImAlexis  I've tried implementing it. But it's showing the part of the div out of place: 
![image](https://user-images.githubusercontent.com/1331394/37783608-3898ac38-2df6-11e8-85ee-c34c5041dae2.png)

Any idea?

Reply:
I'll have to check but I think I added classes to the input so you don't need them on the `<file-browser>`. Check the component file.

Reply:
tnx that was it.